### PR TITLE
Adds 'NativeFunctionInvocationFixer' to be sure a leading slash will …

### PR DESCRIPTION
**Pull request by sd-lueckel**
…be added to internal functions (speeds up code for opcache)

### CODE REVIEW

**Review by michz — DISMISSED** [review]

File: tests/CleanCode.php [hunk 1]
```diff
@@ -28,4 +28,10 @@ public function thisReadsPrivateClassMember(): string
     {
         return $this->privateClassMember;
     }
+
+    public function thisUsesAInternalFunction(): string
```
Comment (michz) [inline tests/CleanCode.php]:
```suggestion
    public function thisUsesAnInternalFunction(): string
```

**Review by michz — CHANGES_REQUESTED** [review]

File: tests/CleanCode.php [hunk 2]
```diff
@@ -28,4 +28,10 @@ public function thisReadsPrivateClassMember(): string
...
+    {
+        // Uses '\md5' instead of 'md5' to improve opcode handling
+        return \md5('hashMe');
```
Comment (twojtylak) [inline tests/CleanCode.php:35]: Ist der Test wirklich richtig?

**Review by michz — APPROVED** [review]

File: tests/CleanCode.php (hunk 2 above)
Comment (sd-lueckel) [reply tests/CleanCode.php:35]: Yes, the 'test' only checks if the class passes the coding standards. In the future we should try to use it the other way around. But for the moment it is ok

Comment (twojtylak) [reply tests/CleanCode.php:35]: ah okay ...

**Review by twojtylak — APPROVED** [review]

**Pull request merged**